src/lib/components/auth-cards/CheckEmailCard.svelte

```diff
@@ -45,10 +45,20 @@
 
   <div class="modal-body-content" slot="card-body">
     <p class="body-text-info">{body}</p>
-    <p class="body-text-action">
+    {#if !checkPW}
+      <p class="body-text-action">
+        Ready to sign in?<button
+          on:click={() => {
+            handleTrigger("welcome");
+          }}>Sign in</button
+        >
+      </p>
+    {/if}
+    <!-- <p class="body-text-action">
```

Comment:
Same question wrt commented-out code, if we're going to use it in the near future it's fine, but otherwise I'd just delete it.